.github/workflows/ci.yml

```diff
@@ -301,17 +301,30 @@ jobs:
           workload_identity_provider: projects/345757944225/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions-provider
           service_account: github-actions-gcr@sentryio.iam.gserviceaccount.com
 
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v1'
+        with:
+          # https://github.com/google-github-actions/auth#authenticating-via-workload-identity-federation
+          # You must use the Cloud SDK version 390.0.0 or later to authenticate the bq and gsutil tools.
+          version: '>= 390.0.0'
+
       - name: Configure docker
         run: |
           gcloud auth configure-docker us.gcr.io
 
-      - name: Push to us.gcr.io
+      - name: Push to us.gcr.io and upload deployment assets
         run: |
           set -euxo pipefail
           docker pull -q "$IMG_VERSIONED"
           docker tag "$IMG_VERSIONED" "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}"
           docker push "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}"
 
+          docker run --rm --entrypoint cat "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}" /opt/relay-debug.zip > relay-debug.zip
+          docker run --rm --entrypoint cat "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}" /opt/relay.src.zip > relay.src.zip
+          docker run --rm --entrypoint tar "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}" -cf - /lib/x86_64-linux-gnu > libs.tar
```

Comment:
Since `"us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}"` repeats so many times right now, it would be a good opportunity to refactor this a bit and create an env variable (which makes all of this a bit shorter), see how `IMG_VERSIONED` was configured above (line 294).